src/config/config.js

```diff
@@ -29,7 +29,17 @@ function getContractMetadata(versionNum = Config.CONTRACT_VERSION_NUM, testnet =
   return mainnetMetadata[versionNum];
 }
 
+function getBlockChainConstants(testnet=Config.TESTNET) {
+  const BLOCK_0_TIMESTAMP = 1504695029; // same for testnet & mainnet
```

Comment:
can we put this in the `Config` obj since it is the same for testnet and mainnet?

Reply:
or if you wanted to keep this separate constants, why not just assign it as a obj instead of a func?

Reply:
done